include/_mgp_mock.py

```diff
@@ -0,0 +1,183 @@
+import typing
+
+import networkx as nx
+
+
+class ImmutableObjectError(Exception):
+    pass
+
+
+class LogicErrorError(Exception):
+    pass
+
+
+class Vertex:
+    __slots__ = ("_id", "_graph")
+
+    def __init__(self, id: int, graph: nx.MultiDiGraph) -> None:
+        if not isinstance(id, int):
+            raise TypeError(f"Expected 'int', got '{type(id)}'")
+
+        if not isinstance(graph, nx.MultiDiGraph):
+            raise TypeError(f"Expected 'networkx.classes.multidigraph.MultiDiGraph', got '{type(graph)}'")
+
+        if not graph.has_node(id):
+            raise IndexError(f"Unable to find vertex with ID {id}.")
+
+        self._id = id
+        self._graph = graph
+
+    @property
+    def id(self) -> int:
+        return self._id
+
+    @property
+    def graph(self) -> nx.MultiDiGraph:
+        return self._graph
+
+    @property
+    def labels(self) -> typing.List[int]:
+        return self._graph.nodes[self._id]["label"].split(":")
```

Comment:
Can we make "label" a constant? This is because nx doesn't have its own label store right?

Reply:
That’s right, networkx does not have its own label store. I’ll rename the field.